src/OrchardCore.Modules/OrchardCore.ContentFields/Services/DefaultContentPickerResultProvider.cs

```diff
@@ -39,7 +39,7 @@ public async Task<IEnumerable<ContentPickerResult>> Search(ContentPickerSearchCo
                 results.Add(new ContentPickerResult
                 {
                     ContentItemId = contentItem.ContentItemId,
-                    DisplayText = contentItem.DisplayText
+                    DisplayText = contentItem.ToString()
```

Comment:
We should flow the ContentItem itself down to the view, as we are already loading them in memory, and let the field view call ToString(). Some other Editors might render it differently (like using BuildDisplay)

Reply:
I see what you mean when it comes to `ContentPickerItemViewModel`. That shouldn't be needed anymore I don't think. But it looks like you're referring to usage of the `ContentPickerResult` object.

Are you saying that `ContentPickerResult` provider's signature should be changed to contain a `ContentItem` property, or that we should forgo the `ContentPickerResult` type altogether and return a `ContentItem` by itself from `ContentPickerResultProvider`? - either would require changes to at least `LuceneContentPickerResultProvider`, to load the content item from the document, which it currently doesn't need to do.

Reply:
Ok, didn't know it was not loading the content item, so all is good then